trusttrees/draw.py

```diff
@@ -166,5 +168,17 @@ def generate_graph(
         if open_graph_file:
             print('[ STATUS ] Opening final graph...')
             subprocess.call(['open', filename])
+        if upload_args:
+            print('[ STATUS ] Uploading to AWS...')
+            prefix, bucket = upload_args.split()
```

Comment:
Should we do `.split(',')`?
I might be wrong, but if `upload_args` is a string, e.g. `'some_prefix/,bucket'` this seems needed, I may be rusty with argparse though.

Reply:
Ah you're right, I did that when I was testing but forgot to add it in to the actual file